package.json

```diff
@@ -330,11 +335,6 @@
         "command": "list.collapse",
         "when": "listFocus && !inputFocus"
       },
-      {
-        "key": "g g",
```

Comment:
We need this one. Mind putting it back?

Reply:
After removing this, `gg` seems to work.

I'm new to VSCode extension development and I'm not used to package.json and what should I write to `contributes.commands`. so please correct me if I'm wrong.
But unlike other VSCode extensions, I guess it's enough to lists only one key sequences here to send to VSCodeVim's state machine.
The state machine processes keys one by one, and fire the command if key sequence is matched.